api_tests/package.json

```diff
@@ -32,7 +32,7 @@
         "bitcoin-core": "^3.0.0",
         "bitcoinjs-lib": "^5.1.10",
         "chmod": "^0.2.1",
-        "comit-sdk": "^0.17.0",
+        "comit-sdk": "comit-network/comit-js-sdk#0df2d5a0",
```

Comment:
You could use https://github.com/whitecolor/yalc if you wanted to.

Reply:
Would you use that in the comit-rs CI?